src/cmd/create.go

```diff
@@ -90,6 +91,12 @@ func init() {
 		"",
 		"Create a toolbox container for a different operating system release than the host")
 
+	flags.StringVarP(&createFlags.shell,
+		"shell",
```

Comment:
It will be better not to add a new flag to `toolbox create`. Instead, I would let users configure it separately through `usermod --shell` or `chsh` inside the container. Those are easy enough to do and well understood, and we can avoid exposing an extra user-facing interface that we would have to maintain forever.

The main part is to somehow get `toolbox enter` to actually use the default shell specified for the user inside the container, instead of always using what's in the `SHELL` environment variable on the host.

Reply:
@yann-soubeyrand has a work in progress at https://github.com/containers/toolbox/pull/1037